parsers/mysql/mysql.go

```diff
@@ -314,6 +315,14 @@ func (p *Parser) ProcessLines(lines <-chan string, send chan<- event.Event) {
 	var foundStatement bool
 	groupedLines := make([]string, 0, 5)
 	for line := range lines {
+		// mysql parser does not support capturing fields in the line prefix - just
+		// strip it.
+		if prefixRegex != nil {
+			var prefix string
+			prefix, _ = prefixRegex.FindStringSubmatchMap(line)
```

Comment:
if you're not using the `-Map` functionality of this variant, let's just use `prefixRegex.FindString` directly.